opencti-platform/opencti-graphql/src/database/file-storage.js

```diff
@@ -366,73 +369,33 @@ export const upload = async (context, user, path, fileUpload, opts) => {
     }
   });
   await s3Upload.done();
+  existingFile = await loadFile(user, key);
   const file = {
     id: key,
     name: filename,
-    size: readStream.bytesRead,
+    size: existingFile.size,
     information: '',
     lastModified: new Date(),
     lastModifiedSinceMin: sinceNowInMinutes(new Date()),
     metaData: { ...fullMetadata, messages: [], errors: [] },
     uploadStatus: 'complete'
   };
+  // Register in elastic
+  await indexFileToDocument(file);
```

Comment:
we should probably think about elastic mapping for this object (which fields will we need to query, and which can we just store, for example metaData could be a flat object)
and maybe we don't need to store everything, for example "information" is empty

Reply:
As we do a lot of search/filtering on the inside elements, we unfortunatly need a concrete object
However you right defining mapping can still be a good idea